starter-code/js/app.js

```diff
@@ -1,6 +1,66 @@
 /* setup your angular app here */
-
+var app = angular.module('FoodSort', []);
 //debug stuff to show the app is loading and fruit / veggies are available
 console.log('App Started');
 console.log('Fruit count', fruit.length);
-console.log('Veggie count', vegetables.length);
\ No newline at end of file
+console.log('Veggie count', vegetables.length);
+
+app.controller('FoodCtrl', ['$scope', function($scope) {
+
+  $scope.fruit = [
```

Comment:
Since we have access to the fruit and vegetables arrays defined in test.js, let's assign those to ```$scope.fruit``` and ```$scope.vegetables```.
```
$scope.fruit = fruit;
$scope.vegetables = vegetables;
$scope.pool = fruit.concat(vegetables);
```

Also, you are overwriting ```$scope.fruit``` and ```$scope.vegetables``` and making them empty arrays on lines 24 & 25, so be sure to delete these or you will have nothing in your columns! ;)